app/src/main/java/ch/epfl/sweng/rps/models/User.kt

```diff
@@ -4,33 +4,37 @@ package ch.epfl.sweng.rps.models
  *
  */
 data class User(
-    val username: String?,
+    val username: String? = "",
 
-    val uid: String,
+    val uid: String = "",
 
-    val gamesHistoryPrivacy: Privacy,
+    val gamesHistoryPrivacy: String = "",
 
-    val hasProfilePhoto: Boolean,
+    val hasProfilePhoto: Boolean = false,
 
-    val email: String?
+    val email: String? = "",
+
+    val matchesList: List<String>? = null
```

Comment:
matches list is a history?

Reply:
Yes, it is a list of matchesID, putting directly the matches is not optimal as the user might not need them, the matches data are loaded based on the IDs of the list on request.